src/csharp/Microsoft.Spark/Interop/Scala/Option.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Spark.Interop.Ipc;
+
+namespace Microsoft.Spark.Interop.Scala
```

Comment:
Was it supposed to be `Microsoft.Spark.Interop.Scala` or `Microsoft.Spark.Internal.Scala`

Reply:
It should have been `Microsoft.Spark.Interop.Internal.*` I moved the existing ones as well.